content/en/tracing/setup_overview/open_standards/otlp_ingest_in_the_agent.md

```diff
@@ -0,0 +1,233 @@
+
+
+[1]: /agent/docker/
+[2]: /agent/docker/apm/#docker-network
+{{% /tab %}}
+{{% tab "Kubernetes (Daemonset)" %}}
+
+1. Follow the [Kubernetes Agent setup][1].
+
+2. Configure the following environment variables in both the trace Agent container and the core Agent container:
+   
+   For gPRC:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT # enables gRPC receiver on port 4317
+   value: "0.0.0.0:4317"
+   ```
+
+   For HTTP:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT # enables HTTP receiver on port 55681
+   value: "0.0.0.0:55681"
+   ```
+3. Map the container ports 4317 or 55681 to the host port for the trace Agent container:
+
+   For gPRC:
+   ```
+   ports:
+     - containerPort: 4317
+       hostPort: 4317
+       name: traceportgrpc
+       protocol: TCP
+   ```
+
+   For HTTP 
+   ```
+   ports:
+     - containerPort: 55681
+       hostPort: 55681
+       name: traceporthttp
+       protocol: TCP
+   ```
+
+4. In the application deployment file, configure the endpoint that the OpenTelemetry client sends traces to with the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable:
+
+   For gPRC:
+   ```
+   env:
+    - name: <DD_AGENT_HOST>
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: "http://$<DD_AGENT_HOST>:4317" # sends to gRPC receiver on port 4317
+   ```
+
+   For HTTP:
+   ```
+   env:
+    - name: <DD_AGENT_HOST>
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: "http://$<DD_AGENT_HOST>:55681" # sends to HTTP receiver on port 55681
+   ```
+
+
+[1]: /agent/kubernetes/?tab=daemonset
+{{% /tab %}}
+{{% tab "Kubernetes (Helm)" %}}
+
+1. Follow the [Kubernetes Agent setup][1].
+
+2. Set environment variables for the Agent. You can either use `set` commands:
+
+   For gRPC:
+   ```
+   --set 'agents.containers.traceAgent.ports[0].containerPort=4317,agents.containers.traceAgent.ports[0].hostPort=4317,agents.containers.traceAgent.ports[0].name=traceportgrpc,agents.containers.traceAgent.ports[0].protocol=TCP' 
+
+   --set "datadog.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT,datadog.env[0].value=0.0.0.0:4317"
+   ```
+   For HTTP:
+   ```
+   --set 'agents.containers.traceAgent.ports[0].containerPort=55681,agents.containers.traceAgent.ports[0].hostPort=55681,agents.containers.traceAgent.ports[0].name=traceporthttp,agents.containers.traceAgent.ports[0].protocol=TCP'
+
+   --set "datadog.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT,datadog.env[0].value=0.0.0.0:55681"
+
+   ```
+
+   Or in the set them in the `datadog.env` file (a more flexible but less automated way than the `set` commands):
```

Comment:
```suggestion
   Or set them in the `datadog.env` parameter of the `values.yaml` file:
```
I removed the part about "more flexible" as I believe that both methods should be equivalent, with the value in `--set` taking precendence over the value set in the user-supplied `values.yaml` file (https://helm.sh/docs/chart_template_guide/values_files/).